python/cuml/ensemble/randomforestclassifier.pyx

```diff
@@ -520,7 +520,8 @@ class RandomForestClassifier(Base):
             raise ValueError("The number of columns/features in the training"
                              " and test data should be the same ")
 
-        preds = np.zeros(n_rows, dtype=np.int32)
+        preds = cuda.device_array(n_rows,
```

Comment:
Does this work if the forest is float64? Maybe dtype = self.dtype?

Reply:
its suppose to be int32 since preds are labels for classification